method-info/totals-and-components/totals-and-components.md

```diff
@@ -0,0 +1,145 @@
+        (732),
+        (101),
+        (165)
+    ],
+    amend_total=False,
+    predictive=1689,
+    precision=10,
+    auxiliary=None,
+    absolute_difference_threshold=28,
+    percentage_difference_threshold=0.1
+)
+
+# The output will be returned as an object.
+# You will need to destructure the object to extract the values and
+# one way of doing this is using the built-in function vars().
+# vars() is used to return the __dict__attribute for the specified
+# module, class, instance or any other object with a
+# __dict__attribute
+print(vars(result))
+```
+
+Running this command will then give you the results from the totals and components area.
+
+### Example Output
+
+The output data is determined by the TCC marker. Some values would be returned as None if they are not calculated.
+The output is as follows:
+
+| identifier | absolute_difference | lower_percentage_threshold | upper_percentage_threshold | final_total | final_components | tcc_marker |
+| --- | --- | --- | --- | --- | --- | --- |
+| 1 | 59 | 1467 | 1793 | 1689 | [654.8760735,758.4957055,104.6558282,170.9723927] | C | <!-- Components have been corrected  -->
+| 2 | 11 | None | None | 11 | [7,0,2,2] | "T" | <!-- Total value has been corrected -->
+| 3 | None | None | None | 11 | [0,0,0,0] | "S" |  <!-- Method has stopped and no outputs returned -->
+| 4 | None | 9729.9 | 11892.1 | 10811 | [9201,866,632,112] | "N" | <!-- No correction has been applied -->
+| 5 | None | 9729.9 | 11892.1 | 12492 | [9201,866,632,112] | "M" | <!-- Manual editing is required -->
+
+The breakdown of the received outputs are as follows:
+
+* Unique Identifier – Any e.g., Business Reporting Unit
+* Absolute difference - the absolute difference between the predictive value and the sum of the original components
+* Lower percentage threshold - the lower threshold calculated for the percentage range
+* Higher percentage threshold - the higher threshold calculated for the percentage range
+* Final total - the final total will be corrected if applicable or will remain as the original if not
+* Final component - the final components will be corrected if applicable, if not it will remain as original components
+
+### Example Usage
+
+There are various ways of running this method. You can find more information regarding this on totals and components [documentation page](https://github.com/ONSdigital/sml-python-small/blob/main/sml_small/editing/totals_and_components).
+
+## Test Data
+
+The test data mentioned in the example above can be found alongside this user documentation.
+
+## Additional Information
+
+The ONS Statistical Methods Library website [methods page](https://statisticalmethodslibrary.ons.gov.uk/methods) contains further information about the methods including:
+
+* a methodological specification, which contains further detail about the mathematical definition of the method algorithm
+* a link to the github repository which contains detailed API information as part of the method code
```

Comment:
**and** a link to the github repository..

Reply:
Done